src/redteam/plugins/mathPrompt.ts

```diff
@@ -0,0 +1,77 @@
+import dedent from 'dedent';
```

Comment:
let's throw a comment in this file referencing https://arxiv.org/html/2409.11445v1